src/FQDN.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Etsy\Opsweekly;
+
+use \InvalidArgumentException;
```

Comment:
I was originally trying to handle errors with the regex, but it turned out to be tricky to test for. I'll remove it.